src/helpers/figures/charts/chart_factory.ts

```diff
@@ -184,6 +187,18 @@ export function getSmartChartDefinition(zone: Zone, getters: Getters): ChartDefi
       legendPosition: newLegendPos,
     };
   }
+  const _dataSets = createDataSets(getters, dataSets, sheetId, dataSetsHaveTitle);
+  if (singleColumn && getData(getters, _dataSets[0]).every((e) => typeof e === "string")) {
```

Comment:
why checking it's all string here but down below also check that it's not an error?